config/prometheus/monitor.yaml

```diff
@@ -10,11 +10,7 @@ metadata:
 spec:
   endpoints:
```

Comment:
we should not change the existing montor.yaml, I have created another file for insecure, maybe production we need to revert this change